integration/setup/setup.go

```diff
@@ -342,3 +327,68 @@ func insertBenchmarkProvider(tb testing.TB, ctx context.Context, collection *mon
 
 	return
 }
+
+// cleanupCollection cleans up the database if `opts.DatabaseName` is empty,
```

Comment:
One of the reasons we write comments, even for unexported functions, is to catch bad design. But there we have both bad design and description that normalize this bad design, no?

I think the "cleanup collection" function should not clean up the database, just as it should not make coffee, for example.